README.md

```diff
@@ -49,16 +30,17 @@ scoop install ffmpeg
 - Beam Size: Adjust the amount of transcriptions generated and considered, which affects accuracy and transcription generation time.
 - Transcription Method: Choose "real-time" for real-time diarization and transcriptions, or "sequential" for periodic transcriptions with more context.
 
-## Latency
-If you want minimal latency, use the real-time mode. If you don't mind growing latency and prioritize accuracy, use the sequential mode.
+## Troubleshooting
+
+- If you're unable to connect from the client to the server, use an ngrok tunnel to expose port 8000.
```

Comment:
Why would this happen?

Reply:
Honestly, no idea. Worked fine on MacOS, but didn't work on Windows. If I had to take a guess, I'd say the connection is blocked.

Reply:
> Honestly, no idea. Worked fine on MacOS, but didn't work on Windows. If I had to take a guess, I'd say the connection is blocked.

After changing http://0.0.0.0:8000/ to http://localhost:8000/ in App.js, it is able to run.

Reply:
@Epresin Good find! Appreciate it :)
Seems to work just fine on MacOS as well, so it's probably the safer bet :)